### PR TITLE
feat: user given metric summary 

### PR DESCRIPTION
**Pull request by thapasya-m**
ISSUES CLOSED: 2001

### Your checklist for a pull request to sitespeed.io
Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [X] I'm making a big change or adding functionality so I've already opened an issue proposing the change to other contributors, so I got feedback on the idea before took the time to write precious code
- [x] Check that your change/fix has corresponding unit tests (if applicable)
- [X] Squash commits so it looks sane
- [ ] Update the documentation https://github.com/sitespeedio/sitespeed.io/tree/master/docs in another PR
- [X] Verify that the test works by running `npm test` and test linting by `npm run lint`

### Description
Please describe your pull request and tell us the fix #
2001
Thank you for making sitespeed.io even better!

### CODE REVIEW

**Comment by soulgalore** [comment]
Hi @thapasya-m wow looks super cool!

A couple of small things: 
* Add a test in `.travis.yml`so we run through the code path., You can just use one of the current tests and add the new parameter with a configuration
* Add another PR with updated docs, I think you can add a section in https://www.sitespeed.io/documentation/sitespeed.io/configuration/#the-basics so that there are two different examples.
* When we display size, can we squeeze in **(kb)** in the header? 
* I think should try to follow the the same structure of default column that we show today but we have different columns depending if you run in Docker or not so ... let us discuss if there's a clean way to fix that.

This will be great addition, thank you again @thapasya-m ! After this we should add more friendly names and then some kind of display name. 
Best
Peter

File: lib/cli/cli.js [hunk 1]
```diff
@@ -1142,6 +1173,21 @@ module.exports.parseCommandLine = function parseCommandLine() {
         'Will add a link on the waterfall page, helping you to compare the HAR. The full path to your compare installation. In the format of https://compare.sitespeed.io/',
       group: 'HTML'
     })
+    .option('html.pageSummaryMetrics', {
+      type: 'array',
+      describe:
+        'Select from the list of metrics to be tested for the given URL(s)',
```
Comment (soulgalore) [inline lib/cli/cli.js]: Can you squeeze in a full example here? Users usually have a problem understanding the difference between an array/string in the context of the cli, I've seen so many different solutions/tries :)

**Comment by soulgalore** [comment]
@thapasya-m I think there's some kind of missmatch between metrics/headers in some cases. I tried using docker:
```
docker build -t sitespeedio/sitespeed.io .
docker run --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io https://www.sitespeed.io/
```

And then the page looks like this:
Screenshot 2019-10-23 at 09 05 44

**Comment by soulgalore** [comment]
Else it getting close now, great work @thapasya-m :)

File: lib/cli/cli.js (hunk 1 above)
Comment (thapasya-m) [reply lib/cli/cli.js]: ok, Peter.

**Comment by soulgalore** [comment]
Thank you @thapasya-m :)

**Pull request merged**